src/components/Search.js

```diff
@@ -2,30 +2,26 @@ import { Form } from "./uploads/uploadStyles"
 import { useState } from "react"
 // import { URL } from "./photolist/PhotoLists"
 
-const Search = () => {
-    const [searchWord, setSearchWord] = useState("")
+const Search = ({ onSubmit }) => {
+    const [value, setValue] = useState("")
     // const [filteredResults, setFilteredResults] = useState([]);
 
-    const getSearchedValue = (e) => {
-        setSearchWord(e.target.value)
+    const handleSubmit = (e) => {
+        e.preventDefault()
+
+        onSubmit(value)
+    }
 
-        // if (searchWord !== '') {
-        //     const filteredData = photos.filter((item) => {
-        //         return Object.values(item).join('').toLowerCase().includes(searchWord.toLowerCase())
-        //     })
-        //     setFilteredResults(filteredData)
-        // }
-        // else{
-        //     setFilteredResults(photos)
-        // }
+    const getSearchedValue = (e) => {
```

Comment:
`handleSearchValueChange` is a better name

Reply:
okay thanks
I will do that now